plugins/github-actions/src/api/GithubActionsApi.ts

```diff
@@ -29,60 +29,60 @@ export const githubActionsApiRef = createApiRef<GithubActionsApi>({
 
 export type GithubActionsApi = {
   listWorkflowRuns: ({
-    token,
+    hostname,
     owner,
     repo,
     pageSize,
     page,
     branch,
   }: {
-    token: string;
+    hostname?: string;
```

Comment:
Should be possible to make this required, as any existing usage would just ignore it